### PR TITLE
[Stabilization 2.6.0] NearMenuExamples: 1m auto follow example toggle pin state is not updated when auto follow is triggered.

### PR DESCRIPTION
**Pull request by cre8ivepark**
## Overview
[Stabilization 2.6.0] NearMenuExamples: 1m auto follow example toggle pin state is not updated when auto follow is triggered.
Fix: Added 'Auto Follow Trigger' event to the FollwMeToggle script. Toggle Pin button using this event properly.

## Before: Pin button is not toggled on auto follow
https://user-images.githubusercontent.com/13754172/108447107-26c09600-7214-11eb-9fcc-becaf748244d.mp4

## After: Pin button is properly toggled on auto follow
https://user-images.githubusercontent.com/13754172/108447172-46f05500-7214-11eb-9087-ad94436d6621.mp4

2021-02-18 17_59_12-MRTK260Stabilization - Microsoft Visual Studio

## Changes
- Fixes: #9335

### CODE REVIEW

File: Assets/MRTK/SDK/Features/UX/Scripts/Slate/FollowMeToggle.cs [hunk 1]
```diff
@@ -117,6 +118,13 @@ public Transform AutoFollowTransformTarget
         private RadialView radialView = null;
         private Coroutine autoFollowDistanceCheck = null;
 
+        [Header("Events")]
+        /// <summary>
+        /// Event that gets fired when auto follow is triggered
+        /// </summary>
+        public UnityEvent AutoFollowTriggered = new UnityEvent();
```
Comment (keveleigh) [inline Assets/MRTK/SDK/Features/UX/Scripts/Slate/FollowMeToggle.cs:125]: [We should make this a private field](https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/Contributing/CodingGuidelines.html#encapsulation) with the `SerializeField` tag, with a property accessor if needed

Comment (cre8ivepark) [reply Assets/MRTK/SDK/Features/UX/Scripts/Slate/FollowMeToggle.cs]: Updated. Thanks!

**Review by Cameron-Micka — APPROVED** [review]
Looks great!

**Pull request merged**